analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Invocations.cs

```diff
@@ -22,6 +22,7 @@
 using SonarAnalyzer.SymbolicExecution.Constraints;
 using SonarAnalyzer.SymbolicExecution.Roslyn;
 using SonarAnalyzer.UnitTest.TestFramework.SymbolicExecution;
+using StyleCop.Analyzers.Lightup;
```

Comment:
Somewhere in the history, there was a mistake with file names.
This one is `RoslynSymbolicExecutionTest.Invocations.cs` and we also have `RoslynSymbolicExecutionTest.Invocation.cs`.

Let's merge them to `Invocation` (singular) variant. In this or a follow up PR